api/src/opentelemetry/trace/BUILD

```diff
@@ -0,0 +1,22 @@
+# Copyright 2019, OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+package(default_visibility = ["//visibility:private"])
+
+cc_library(
+    name = "trace",
+    srcs = ["span_id.cc"],
+    visibility = ["//visibility:public"],
+    deps = ["@io_opentelemetry_cpp//api/include/opentelemetry/trace"],
+)
```

Comment:
Do you think it would be worth while to use customized wrapper `cc_library`, `cc_binary`, etc functions like [Envoy](https://github.com/envoyproxy/envoy/blob/master/bazel/envoy_library.bzl#L33) does in case we wanted to apply option changes across all of the project's cc_library's?

Reply:
I'd prefer to avoid it for now but it might become inevitable. In opencensus-cpp, we set `copts = DEFAULT_COPTS` on every cc_library.